src/tests/globalStyles.e2e.ts

```diff
@@ -32,7 +32,7 @@ describe("global styles", () => {
             opacity: opacity
           };
```

Comment:
@caripizza Do these tests need to use `notice`? I originally thought they were testing component-specific animation, but it looks like the tests focus on asserting values on the global classes.

Reply:
Nope, please remove or improve them (these were just tests I added with the [original demo page](https://github.com/Esri/calcite-components/blob/master/src/demos/animation/animation.html)).

It seems more important to test that animations can be disabled globally by setting the timing var to `0s`.